src/main/java/seedu/address/logic/commands/property/FilterPropertiesCommand.java

```diff
@@ -26,18 +26,18 @@ public class FilterPropertiesCommand extends Command {
 
     public static final String COMMAND_WORD = "filterprops";
 
-    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Finds all properties in the database that"
-            + " either has a price that is within the given price range, contains all characteristics in the given"
-            + " characteristics list, or has the given seller. You can filter by more than one criteria at a time.\n"
-            + " Pass in " + PREFIX_FUZZY + " for fuzzy filtering, i.e. filtered properties will only need to satisfy"
-            + " at least one of the conditions specified. \n"
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Filters all properties in Cobb that satisfy the given"
+            + " price range AND contains all the given characteristics AND has the given owner name.\n"
```

Comment:
same as above: if the user doesn't provide all three conditions then this explanation might confuse them